packages/checkbox/src/CheckboxPresenter/CheckboxPresenter.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import FormInput from "../presenters/FormInput";
+
+import icons from "@hig/icons";
```

Comment:
Per our lint rules, absolute imports should be before relative imports, and separated by a line break.